metrotranscom/doorway
# feat: paper applications - new fields

This PR addresses [#(750)](https://app.zenhub.com/workspaces/bloom-5dc32d7144bd400001315dac/issues/gh/metrotranscom/doorway/750)


## Description

Adding ReceivedAt and ReceivedBy fields to application table. Present on the Partners Portal when creating/updating/viewing paper applications; ReceivedAt follows logic for SubmissionDate. Appears in the application export csv.

## How Can This Be Tested/Reviewed?

Login to Partners Portal as an admin or a partner.
Navigate to a listing's application tab.
Add an application, fill out `Date Received At`, `Time Received At` and `Received By` fields.
Submit application, verify data appears correctly in details view.
Edit application, verify data appears correctly, edit data, submit and verify change.
Go back to the application tab.
Export application csv and verify new fields are present and contain correct data.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
